cli/version/version_match.go

```diff
@@ -0,0 +1,161 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/tarantool/tt/cli/util"
+)
+
+type requiredField uint8
+type requiredFields []requiredField
+
+const (
+	// Values according order of Fields in regex from `matchVersionParts()`.
+	requiredFieldBuildName requiredField = iota
+	requiredFieldMajor
+	requiredFieldMinor
+	requiredFieldPatch
+	requiredFieldReleaseType
+	requiredFieldReleaseNum
+	requiredFieldAdditional
+	requiredFieldHash
+	requiredFieldRevision
+
+	countOfRequiredFields
+)
+
+const NotFoundVersion = "NOT_FOUND"
+
+// exploreMatchVersion examines the given string and retrieves the version components,
+// that define the rules for checking for version consistency.
+func exploreMatchVersion(verStr string) (Version, requiredFields, error) {
+	var version Version
+	fields := make(requiredFields, 0, countOfRequiredFields)
+
+	matches, err := matchVersionParts(verStr, false)
+	if err != nil {
+		return version, fields, err
+	}
+
+	version.BuildName = matches["buildname"]
+	if version.BuildName != "" {
+		fields = append(fields, requiredFieldBuildName)
+	}
+
+	if matches["major"] != "" {
+		if version.Major, err = util.AtoiUint64(matches["major"]); err != nil {
+			return version, fields, fmt.Errorf("can't parse Major; %w", err)
```

Comment:
In my opinion `:` is more prettier than `;` symbol.

Besides, as I have noticed, there is a pattern in `tt` that the error wrapper starts with the error description and is complemented after `:` by the error received.

Or am I missing something and this wrap has a different purpose?

```suggestion
			return version, fields, fmt.Errorf("can't parse Major: %w", err)
```

Reply:
Done.